wled00/FX.h

```diff
@@ -44,8 +44,20 @@
 
 /* Not used in all effects yet */
 #define WLED_FPS         42
-#define FRAMETIME_FIXED  (1000/WLED_FPS)
 #define FRAMETIME        strip.getFrameTime()
+#if defined(ARDUINO_ARCH_ESP32) && !defined(CONFIG_IDF_TARGET_ESP32C3)    // all ESP32 except -C3(very slow, untested)
+  #define FRAMETIME_FIXED  (strip.getFrameTime() < 10 ? 12 : 24)            // allow faster FRAMETIME_FIXED when target FPS >= 100  
```

Comment:
This counters the **fixed** logic. If it is fixed than it should not vary.

Reply:
> This counters the **fixed** logic. If it is fixed than it should not vary.

I agree - it's only used 3 times in fx.cpp, and it seems to be more like a "reference time" for speedup/slowdown. I'll change it and keep the original fixed value.

Reply:
done.